### DISCUSSION TITLE
ROS2 Jazzy for Debian Bookworm 32-bit OS

### DISCUSSION THREAD

**Discussion by arunser** [Ar-Ray-code/rpi-bullseye-ros2#21] [Q&A]
Is the ROS2 Jazzy deb package available for Bookworm 32-bit OS? I was searching for it for a specific project where I can only use 32-bit OS. If not can you tell me how to build it from source for 32-bit OS?

**Comment by Ar-Ray-code** [comment]
Which board is intended to run which application?
rpi-bullseye-ros2 is intended to run on RaspberryPi 3B, RaspberryPi Zero2W, RaspberryPi 3A/B+, RaspberryPi4 and RaspberryPi5, all of which support 64-bit OS.

I have confirmed that RaspberryPi zero takes more than 1 minute to load the environment to begin with, so we have removed it from support; please consider an alternative such as micro-ros.

Also, 32-bit OSes may have errors due to various missing options. The `.old` directory contains CMakeLists for ros-humble 32bit.

**Reply by arunser** [reply]
I am using Raspberry Pi 4  8GB. I want to run some applications that are only supported by 32-bit OSes. Since I am using Bookworm, is there a 32-bit implementation of Jazzy? 

How can I use the ros-humble 32-bit in the .old directory?
